libs/input-mapping/src/Table/Options/InputTableOptions.php

```diff
@@ -115,56 +97,6 @@ public function getColumnNamesFromTypes(): array
         return array_column($this->definition['column_types'], 'source');
     }
 
-    /**
-     * @return array{
-     *     columns?: array<string>,
-     *     changedSince?: string,
-     *     whereColumn?: string,
-     *     whereValues?: array<string>,
-     *     whereOperator?: string,
-     *     limit?: integer,
-     *     overwrite?: bool,
-     * }
-     */
-    public function getStorageApiExportOptions(InputTableStateList $states): array
```

Comment:
přesunuté sem https://github.com/keboola/platform-libraries/pull/129/files#diff-bb485fa3a6aee1ec3e12a26e328d7a4f0d62ad1958bb0f1ae4f82f821cb84d0cR43 protože spolíhá na to, že je sourceBranchId nastavené